contracts/contracts/Poll.sol

```diff
@@ -142,17 +156,25 @@ contract Poll is Params, Utilities, SnarkCommon, EmptyBallotRoots, IPoll {
       numMessages++;
     }
 
-    // init messageAq here by inserting placeholderLeaf
+    // init chainHash here by inserting placeholderLeaf
     uint256[2] memory dat;
     dat[0] = NOTHING_UP_MY_SLEEVE;
     dat[1] = 0;
 
     (Message memory _message, PubKey memory _padKey, uint256 placeholderLeaf) = padAndHashMessage(dat);
-    extContracts.messageAq.enqueue(placeholderLeaf);
+    // extContracts.messageAq.enqueue(placeholderLeaf);
+    chainHash = NOTHING_UP_MY_SLEEVE;
```

Comment:
That is an initialization value, before any message is published. In theory it can be some random value (5 for example). We used NOTHING_UP_MY_SLEEVE as it is a generally used value in MACI for pseudorandom initializations.